deployments/helm/nvidia-device-plugin/templates/role.yml

```diff
@@ -10,9 +10,41 @@ rules:
   - apiGroups: [""]
     resources: ["nodes"]
     verbs: ["get", "list", "watch"]
+  {{- if .Capabilities.APIVersions.Has "security.openshift.io/v1/SecurityContextConstraints" }}
+  - apiGroups:
+      - security.openshift.io
+    resourceNames:
+      - privileged
+    resources:
+      - securitycontextconstraints
+    verbs:
+      - use
+  {{- end }}
```

Comment:
Does it make sense to shift this to a named template to use here and below? Not a blocker though.

Reply:
Hi @elezar, apologies for so late reply, I'm back to this thread. 
Yes, it looks like a good approach to follow. Should we include as part of this PR? Or can I do it as a secondary one?

Reply:
Hi @cdesiniotis did you had a chance to review this PR?